workshops/rust_text_game/README.md

```diff
@@ -0,0 +1,408 @@
+
+---
+name: "Text Adventure Game in Rust"
+description: "Make a text adventure game while writing your first Rust program!"
+author: "@panos, @fanoulis"
```

Comment:
```suggestion
author: "@panos, @cfanoulis"
```

This is just a quick fix, just I can't seem to make edits as an admin.

Could you please enable this? I'd help @MatthewStanciu and I.

Thanks!

Reply:
@panos since I can neither apply changes